src/models/Expense.model.js

```diff
@@ -1,9 +1,46 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
+      allowNull: false,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    category: {
+      type: DataTypes.STRING,
+    },
+    note: {
+      type: DataTypes.STRING,
+    },
+  },
+  {
+    tableName: 'expenses',
+    updatedAt: false,
+    createdAt: false,
```

Comment:
The `updatedAt` and `createdAt` fields are set to `false`, which means Sequelize will not automatically manage these timestamps. If you need to track when records are created or updated, consider setting these to `true` or adding custom timestamp fields.